spec/mongoid_spec.rb

```diff
@@ -52,7 +52,7 @@
 
     it "disconnects from all active clients" do
       clients.each do |client|
-        expect(client.cluster).to receive(:disconnect!).and_call_original
+        expect(client.cluster).to receive(:close).and_call_original
```

Comment:
I made this change that fix some test failures with `driver-current`, but now `driver-stable` looks to be failing
was `disconnect!` renamed to `close`?